src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -23,11 +23,12 @@ object HttpGoogleServicesDAO {
   // this needs to match a value in the "Authorized redirect URIs" section of the Google credential in use
   val callbackUri = Uri(s"${baseUrl}${callbackPath}")
 
-  // modify these if we need more granular access in the future
-  val gcsFullControl = "https://www.googleapis.com/auth/devstorage.full_control"
-  val computeFullControl = "https://www.googleapis.com/auth/compute"
-  // TODO: remove compute?
-  val scopes = Seq(gcsFullControl,computeFullControl,"profile","email")
+  // the minimal scopes needed to get through the auth proxy and populate our UserInfo model objects
+  val authScopes = Seq("profile","email")
+  // the minimal scope to read from GCS
+  val storageReadOnly = Seq(StorageScopes.DEVSTORAGE_READ_ONLY)
+  // the scopes we request for the end user during interactive login. TODO: remove compute?
+  val userLoginScopes = Seq(StorageScopes.DEVSTORAGE_FULL_CONTROL,"https://www.googleapis.com/auth/compute") ++ authScopes
```

Comment:
My reading here is that user login scopes have not changed.  Can you confirm this?